src/NUnitFramework/framework/Constraints/DictionaryContainsKeyConstraint.cs

```diff
@@ -16,7 +16,7 @@ namespace NUnit.Framework.Constraints
     public class DictionaryContainsKeyConstraint : CollectionItemsEqualConstraint
     {
         private const string ContainsMethodName = "Contains";
-        private bool _isDeprecatedMode = false;
+        private readonly bool _isDeprecatedMode = false;
```

Comment:
A few non-pattern-matching things like this were also updated in this PR, each of which should give more info to the JIT and potentially a small perf boost